ds3-sdk/src/main/java/com/spectralogic/ds3client/utils/collections/LazyIterable.java

```diff
@@ -62,15 +62,15 @@ public LazyIterable(final LazyLoaderFactory<T> lazyLoaderFactory) {
         List<T> getNextValues();
     }
 
-    private class LazyObjectIterator<T> implements Iterator<T> {
+    private class LazyObjectIterator<E> implements Iterator<E> {
```

Comment:
It was complaining about the type being hidden.  My assumption is that it didn't like that we already had a type T defined in the parent class.